daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/BitFunctionsNot.tdml

```diff
@@ -0,0 +1,305 @@
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+
+  </tdml:defineSchema>
+
+  <tdml:parserTestCase name="testIntNot" root="testIntNot" model="BitFunctions">
+    <tdml:document>1|0</tdml:document>
+      <tdml:infoset>
+        <tdml:dfdlInfoset>
+          <ex:testIntNot>
+            <ex:IntNot1>
+              <num>1</num>
+              <value>-2</value>
+            </ex:IntNot1>
+            <ex:IntNot2>
+              <num>0</num>
+              <value>-1</value>
+            </ex:IntNot2>
+          </ex:testIntNot>
+        </tdml:dfdlInfoset>
+      </tdml:infoset>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="testLongNot" root="testLongNot" model="BitFunctions">
+    <tdml:document>1|0</tdml:document>
+      <tdml:infoset>
+        <tdml:dfdlInfoset>
+          <ex:testLongNot>
+            <ex:LongNot1>
+              <num>1</num>
+              <value>-2</value>
+            </ex:LongNot1>
+            <ex:LongNot2>
+              <num>0</num>
+              <value>-1</value>
+            </ex:LongNot2>
+          </ex:testLongNot>
+        </tdml:dfdlInfoset>
+      </tdml:infoset>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="testShortNot" root="testShortNot" model="BitFunctions">
+    <tdml:document>1|0</tdml:document>
+      <tdml:infoset>
+        <tdml:dfdlInfoset>
+          <ex:testShortNot>
+            <ex:ShortNot1>
+              <num>1</num>
+              <value>-2</value>
+            </ex:ShortNot1>
+            <ex:ShortNot2>
+              <num>0</num>
+              <value>-1</value>
+            </ex:ShortNot2>
+          </ex:testShortNot>
+        </tdml:dfdlInfoset>
+      </tdml:infoset>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="testByteNot" root="testByteNot" model="BitFunctions">
+    <tdml:document>1|0</tdml:document>
+      <tdml:infoset>
+        <tdml:dfdlInfoset>
+          <ex:testByteNot>
+            <ex:ByteNot1>
+              <num>1</num>
+              <value>-2</value>
+            </ex:ByteNot1>
+            <ex:ByteNot2>
+              <num>0</num>
+              <value>-1</value>
+            </ex:ByteNot2>
+          </ex:testByteNot>
+        </tdml:dfdlInfoset>
+      </tdml:infoset>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="testUnsignedIntNot" root="testUnsignedIntNot" model="BitFunctions">
+    <tdml:document>2|1</tdml:document>
+      <tdml:infoset>
+        <tdml:dfdlInfoset>
+          <ex:testUnsignedIntNot>
+            <ex:UnsignedIntNot1>
+              <num>2</num>
+              <value>4294967293</value>
+            </ex:UnsignedIntNot1>
+            <ex:UnsignedIntNot2>
+              <num>1</num>
+              <value>4294967294</value>
+            </ex:UnsignedIntNot2>
+          </ex:testUnsignedIntNot>
+        </tdml:dfdlInfoset>
+      </tdml:infoset>
+  </tdml:parserTestCase>
+
+
+  <tdml:parserTestCase name="testUnsignedLongNot" root="testUnsignedLongNot" model="BitFunctions">
```

Comment:
This test isn't right. This is bitNot of unsigned types, so the <value> part of the infoset should be unsigned, but these test expects -3 and -1. Those should be something else that is signed.